lib/index.js

```diff
@@ -198,7 +199,7 @@ function getExperiments(options) {
     if (experiment) {
       results.push({
         variationName: options.variationNamesMap[experimentId],
-        variationId: options.variationIdsMap[experimentId][0],
+        variationId: options.variationIdsMap[experimentId][0].toString(),
```

Comment:
I want variationId to have a consistent data type. For a multivariate experiment, the variationId contains of several IDs in the format of comma separated string. I think it is good to be consistent (also for writing the tests) to also make the variationID a string when there is only one item in the list.